webapp/channels/src/components/admin_console/user_grid/user_grid_name.tsx

```diff
@@ -13,27 +13,27 @@ type Props = {
     user: UserProfile;
 }
 
-export default class UserGridName extends React.Component<Props> {
-    public render = (): JSX.Element => {
-        const {user} = this.props;
-
-        return (
-            <div className='UserGrid_nameRow'>
-                <ProfilePicture
-                    src={Client4.getProfilePictureUrl(user.id, user.last_picture_update)}
-                    status={status}
-                    size='md'
-                />
-
-                <div className='UserGrid_name'>
-                    <span>
-                        {`${user.username} - ${user.first_name} ${user.last_name}`}
-                    </span>
-                    <span className='ug-email'>
-                        {user.email}
-                    </span>
-                </div>
+const UserGridName = ({
+    user,
+}: Props) => {
+    return (
+        <div className='UserGrid_nameRow'>
+            <ProfilePicture
+                src={Client4.getProfilePictureUrl(user.id, user.last_picture_update)}
+                status={status}
```

Comment:
This seems an issue on the original file. Do you mind making the change?
```suggestion
                status={user.status}
```